src/components/Footer/styles.ts

```diff
@@ -1,51 +1,101 @@
+import { AutoColumn } from 'components/layout/Column'
+import block from 'state/block'
```

Comment:
Why were these imported?

Reply:
I have removed it this are not being used